iOS/Flows/Conversation/Cells/MessageCell.swift

```diff
@@ -16,6 +16,11 @@ import UIKit
 /// The root message and replies are stacked along the z-axis, with the most recent reply at the front (visually obscuring the others).
 class MessageCell: UICollectionViewCell {
 
+    // Interaction handling
+    var handleTappedMessage: ((Messageable) -> Void)?
+    var handleDeleteMessage: ((Messageable) -> Void)?
+    private lazy var contextMenuInteraction = UIContextMenuInteraction(delegate: self)
```

Comment:
Adding a context menu directly to the cells require that I make the underlying collection view interactable. Unfortunately that created conflicts with handling cell taps. To get around this I made specific callbacks for the different actions.